web/pdf_presentation_mode.js

```diff
@@ -97,14 +97,14 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * Switches page when the user scrolls (using a scroll wheel or a touchpad)
      * with large enough motion, to prevent accidental page switches.
-     * @param {number} delta - The delta value from the mouse event.
+     * @param {number} delta - The value from the mouse event in page units.
      */
     mouseScroll: function PDFPresentationMode_mouseScroll(delta) {
       if (!this.active) {
         return;
       }
       var MOUSE_SCROLL_COOLDOWN_TIME = 50;
-      var PAGE_SWITCH_THRESHOLD = 120;
+      var PAGE_SWITCH_THRESHOLD = 1.0;
```

Comment:
`delta * MOUSE_WHEEL_ZOOM_FACTOR` - can you explain what does it mean? I'll trying to make code understandable, e.g. `delta` is a "page" unit. I can make PAGE_SWITCH_THRESHOLD a fraction e.g. 0.3, will that work for you better?